Dockerfile-ci

```diff
@@ -0,0 +1,22 @@
+# Tmp Dockerfile for the CI builds. Will be moved to the ci-images repo later 
+ARG GO_VERSION=1.19.9
+FROM golang:${GO_VERSION}-alpine
```

Comment:
Do the tests themselves use alpine? If so then we are running tests with a different version of libc (musl) than what we actually support (glibc). We should probably use the debian/ubuntu based image instead even though it is slightly larger.

Elastic Agent doesn't official support Alpine Linux yet: https://github.com/elastic/elastic-agent/issues/2075